app/lib/router/router.dart

```diff
@@ -1,8 +1,7 @@
 import 'package:acter/common/dialogs/dialog_page.dart';
```

Comment:
I think this file-rename is also confusing. if anything it should be `router/routes.dart`, no?

Reply:
Hmm, I have created another file with similar name which includes our route definitions in ```utils``` directory. This is more like our navigation function logic. How does that sounds?